employee-rostering-frontend/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
You'll probably need `/* eslint-disable header/header */` on the first line of this rule (or add the license header) to make linter happy. Seems like you copied from some intermediate state of optaweb-vehicle-routing because it's there on master.

Reply:
`/* eslint-disable header/header */` didn't work here for some reason, so I disabled all linting only for this file (disabling camel-case worked on a different file, so I don't know why it didn't work; maybe it also needs a `eslint-enable header-header` at the end of file?)